.github/workflows/site.yml

```diff
@@ -1,35 +1,28 @@
-name: Website
+# This file was autogenerated using `zio-sbt` via `sbt generateGithubWorkflow` 
+# task and should be included in the git repository. Please do not edit 
+# it manually. 
 
-env:
-  JDK_JAVA_OPTIONS: -XX:+PrintCommandLineFlags -XX:MaxMetaspaceSize=4G -Xmx4G -Xss8M # JDK_JAVA_OPTIONS is _the_ env. variable to use for modern Java
-  JVM_OPTS: -XX:+PrintCommandLineFlags -XX:MaxMetaspaceSize=4G -Xmx4G -Xss8M # for Java 8 only (sadly, it is not modern enough for JDK_JAVA_OPTIONS)
+name: website
 
 on:
-  push:
-    branches: [master]
   release:
-    types:
-      - published
+    types: [ published ]
 
 jobs:
-  publish:
+  publish-docs:
     runs-on: ubuntu-20.04
     timeout-minutes: 30
-    if: github.event_name != 'pull_request'
     steps:
-      - uses: actions/checkout@v3
-      - name: Setup Java
-        uses: actions/setup-java@v3.6.0
+      - uses: actions/checkout@v3.1.0
         with:
-          distribution: temurin
-          java-version: 8 # publish for 8
-          check-latest: true
-      - name: Install libuv
-        run: sudo apt-get update && sudo apt-get install -y libuv1-dev
-      - name: Set Swap Space
-        uses: pierotofy/set-swap-space@master
+          fetch-depth: 0
+      - name: Setup Scala and Java
+        uses: olafurpg/setup-scala@v13
```

Comment:
I know that this is generated code and it's not a big deal anyway (yet!).

But olafurpg/setup-scala is essentially dead now.
https://github.com/olafurpg/setup-scala/issues/49
I think we should just use `actions/setup-java`. Maybe we should change the code in the generating plugin.

Reply:
Thank you for the feedback, I will change that on the zio-sbt plugin.